samples/apps/github-qna-webapp-react/src/components/setup/ModelConfig.tsx

```diff
@@ -112,10 +136,10 @@ const ModelConfig: FC<IData> = ({
             <InfoLabel
                 info={
                     <div style={{ maxWidth: 250 }}>
-                        Please note this drop down lists all available models, but not all will work as{' '}
-                        {(modelType === ModelType.Completion ? 'a ' : 'an ') + modelTitle[0]} model.{' '}
-                        <Link href="https://platform.openai.com/docs/models"> Click here to learn more </Link>
-                        about the differences between completion and embedding models.
+                        Please note this drop down lists all available models but not all models will work as{' '}
```

Comment:
Is a way for us to pretest models to ensure they can generate embeddings / completions?  Exposing models that won't work seems meh.

Reply:
Implemented a check / pretest on both

- The response from Azure Open AI /models endpoint includes a property called Capabilities we can use to check if a model supports completion and/or embeddings. I filtered the dropdown in Azure Oai config pages using this logic, disabling the deployments from the models that don't have these capabilities. 
- For Open AI configs, when the user selects a value from the dropdown, I deploy a simple POST request to the /completions or /embeddings endpoints to check the model and it will surface a validation error if the request fails.
- 
<img width="663" alt="Screenshot 2023-03-21 at 3 12 06 PM" src="https://user-images.githubusercontent.com/125500434/226733063-a45bfcb3-ac30-48b6-b78e-55ffa32e1f56.png">
<img width="704" alt="Screenshot 2023-03-21 at 3 09 42 PM" src="https://user-images.githubusercontent.com/125500434/226733067-b2a6cbeb-a658-4fcf-aedd-0e4525432d9c.png">
<img width="651" alt="Screenshot 2023-03-21 at 3 12 34 PM" src="https://user-images.githubusercontent.com/125500434/226733070-3ed7e888-ef24-4288-866c-4125a6f4454c.png">
<img width="333" alt="Screenshot 2023-03-21 at 3 12 40 PM" src="https://user-images.githubusercontent.com/125500434/226733132-316cc5c1-48ae-45f1-91a3-1eb6a928f5d3.png">
<img width="323" alt="Screenshot 2023-03-21 at 3 12 48 PM" src="https://user-images.githubusercontent.com/125500434/226733136-4410f747-65a8-4991-a318-4444e93b3714.png">

Reply:
LGTM